shared/presentation/src/commonMain/kotlin/network/bisq/mobile/presentation/ui/components/molecules/OfferCard.kt

```diff
@@ -93,18 +98,83 @@ fun OfferCard(
                         text = offerListItem.bisqEasyOffer.supportedLanguageCodes.joinToString(", ").uppercase(),
                         color = BisqTheme.colors.light1
                     )
+                    BisqGap.H1()
+                    BisqText.smallRegular(
+                        text = offerListItem.formattedPriceSpec,
+                        color = BisqTheme.colors.light1
+                    )
+                }
+                Row(verticalAlignment = Alignment.CenterVertically) {
+
+//                    Row(verticalAlignment = Alignment.CenterVertically) {
+//                        LanguageIcon()
+//                        BisqText.largeRegular(
+//                            text = ": ${offerListItem. }",
+//                            color = BisqTheme.colors.grey1
+//                        )
+//                    }
+                    BisqGap.H1()
+                    // Len: 13 - "300 - 600 USD"
+                    // Len: 17 - "3,000 - 6,000 XYZ"
+                    // Len: 23 - "150,640 - 1,200,312 CRC"
+                    if (offerListItem.formattedQuoteAmount.length < 18) {
+                        BisqText.baseRegular(
+                            text = offerListItem.formattedQuoteAmount,
+                            color = BisqTheme.colors.light1
+                        )
+                    } else {
+                        BisqText.smallRegular(
+                            text = offerListItem.formattedQuoteAmount,
+                            color = BisqTheme.colors.light1
+                        )
+                    }
                 }
             }
+            /*
+            if (!myTrade) {
```

Comment:
please remove dead code

Reply:
This code is kept here, so ChatButton can be added back in future, when required.
Added the same as a comment in the code, so it's clear now.